### PR TITLE
update

### PR DESCRIPTION
**Pull request by Victor-Akinode**
This project demonstrates how to set up a PostgreSQL database using Docker. It includes the creation of a new schema and table, and the loading of data from a CSV file into the Dockerized PostgreSQL instance. The Python script included connects to the database and executes a query to count the number of records in the specified table.

### CODE REVIEW

**Comment by Victor-Akinode** [comment]
This project demonstrates how to set up a PostgreSQL database using Docker. It includes the creation of a new schema and table, and the loading of data from a CSV file into the Dockerized PostgreSQL instance. The Python script included connects to the database and executes a query to count the number of records in the specified table.

File: postgres_docker_init/src/assignment.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import psycopg2
+from psycopg2 import sql
+
+# Database connection parameters
+DB_NAME = "victor_akinode_db"
```
Comment (JesuFemi-O) [inline postgres_docker_init/src/assignment.py]: It's bad practice to hardcode your credentials like this, please move the credentials into a .env file (ensure the file is in your gitignore too!)

**Review by JesuFemi-O — CHANGES_REQUESTED** [review]
Move out the sensitive credentials from assigment.py.... you're building a portfoliio,  you may want to consider renaming the file to something like `main.py` instead

**Review by JesuFemi-O — APPROVED** [review]
👍🏻

File: postgres_docker_init/src/assignment.py (hunk 1 above)
Comment (Victor-Akinode) [reply postgres_docker_init/src/assignment.py]: I have resolved this. Thanks.

**Pull request merged**